manifests/e/Ezoa/ECHOOF/0.1.1/Ezoa.ECHOOF.locale.en-US.yaml

```diff
@@ -0,0 +1,19 @@
+PackageIdentifier: "Ezoa.ECHOOF"
+PackageVersion: "0.1.1"
+PackageLocale: "en-US"
+Publisher: "Ezoa"
+PackageName: "{{name}"
```

Comment:
```suggestion
PackageName: "ECHOOF"
```

Reply:
Ah that's embarassing, I missed a character in my manifest generator. Thank you for pointing that out!